### ISSUE TITLE
QPA2 has no license

### ISSUE THREAD

**Issue by zickgraf** [sunnyquiver/QPA2#47]
GAP now requires a `License` field in `PackageInfo.g`, see https://github.com/gap-system/gap/commit/4db36d786cd5de1184eb20d48b819b0ce70f7b46. QPA2 has no license, and hence the following message is printed when loading it with current GAP master:
```
#E  component `License' must be bound to a nonempty string containing an SPDX ID
```

**Comment by sunnyquiver** [comment]
Issue resolved in commit 6061684.

**Comment by zickgraf** [comment]
Thanks for the quick fix! :-)
